k6/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '3.4'
+
+networks:
+  k6:
+  grafana:
+
+services:
+  influxdb:
+    image: influxdb:1.8
+    #    entrypoint: /bin/sh
```

Comment:
Finally no, the original author left this in case we want to debug what happens inside containers. You can have an explanation at the section "TIP — Debugging Docker Compose Configuration with a Shell" in article https://medium.com/swlh/beautiful-load-testing-with-k6-and-docker-compose-4454edb3a2e3